frontend/src/components/employees/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import employeesStyle from './employees.module.scss';
+
+const Employees = () => (
+  <div className={employeesStyle.employees}>
+    <h1> Employees </h1>
+    <p> Employees component </p>
+  </div>
+);
+
+export default Employees;
```

Comment:
Given the repetition of code is it worth using a HOC for employee, Rota, and summary?

Reply:
There will be no repetition once the components are fully implemented (I think?).
These are just placeholders.

Reply:
I agree for the now yeah they are just placeholders. I think its worth waiting before too early refactoring.

Reply:
Though I think we might get there one day due to styling for the collapsible sidebar

Reply:
do we have any designs for the different pages yet?